documentation/src/main/wiki/Comparators.md

```diff
@@ -79,9 +83,12 @@ The configuration above will trigger three screens collections (desktop, tablet
 <collect>
     <open/>
     <sleep duration="1000"/>
```

Comment:
Thank you for those changes! They are not obvious and it is very good that we have them in docs!
This configuration looks a little bit redundant, but I believe this is what is required, isn't it?